avr/cores/megacommand/MCL/MCLClipBoard.cpp

```diff
@@ -66,11 +66,9 @@ bool MCLClipBoard::copy_sequencer_track(uint8_t track) {
   }
 
   if (track < NUM_MD_TRACKS) {
-    memcpy(&(md_track->seq_data), &mcl_seq.md_tracks[track],
+    memcpy(md_track->seq_data.data(), mcl_seq.md_tracks[track].data(),
```

Comment:
hmm. what's this?

Reply:
copy/paste track is currently broken fyi.

Reply:
prevent unaligned copy from something with vtable to something without one.
MDTrack gets MDSeqTrackData (no vtable)
MDSeqTrack has one, coming from SeqTrack 😱

Reply:
ah okay. so we want to skip copying the vtable. got you.

Reply:
28     memcpy(mcl_seq.md_tracks[track].data(), md_track->seq_data.data(),
129            sizeof(md_track->seq_data));

isn't sizeof going to  be incorrect then?

Reply:
md_track->seq_data has the correct size (😱)

Reply:
not sure why this has broken copy/paste.

Reply:
```
func_call: bool MCLClipBoard::copy_sequencer_track(uint8_t)
func_call: bool MCLClipBoard::open()
failed write
6
6
func_call: bool MCLClipBoard::paste_sequencer_track(uint8_t, uint8_t)
func_call: bool MCLClipBoard::open()
failed read
```

unrelated to this change.